<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>655</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reconocimiento Python IA - Dante Zulli 7mo 2da</string>
  </property>
  <widget class="QWidget" name="frPrincipal">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frSuperior">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color:rgb(0, 0, 0)
}

QPushButton{
font: 11pt &quot;Calibri&quot;;
background-color: rgb(8, 20, 80);
color:rgb(255, 255, 255);
border-radius:3px;
border: 1px solid white;
}

QPushButton:hover{
font: 11pt &quot;Calibri&quot;;
background-color: rgb(255, 255, 255);
color:rgb(0, 0, 0);
border-radius:3px;
border: 1px solid white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnMenu">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>   MENU</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>GUI_Imagenes/menu.png</normaloff>GUI_Imagenes/menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spEspaciador">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frInferior">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frDesplegable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>100000</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color:rgb(0, 0, 0)
}

QPushButton{
font: 11pt &quot;Calibri&quot;;
background-color: rgb(8, 20, 80);
color:rgb(255, 255, 255);
border-radius:3px;
border: 1px solid white;
}

QPushButton:hover{
font: 11pt &quot;Calibri&quot;;
background-color: rgb(255, 255, 255);
color:rgb(0, 0, 0);
border-radius:3px;
border: 1px solid white;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="lblTitulo">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 75 bold 16pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Reconocimiento:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnObjetos">
            <property name="text">
             <string>Objetos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnContornos">
            <property name="text">
             <string>Contornos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFacial">
            <property name="text">
             <string>Facial</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnManos">
            <property name="text">
             <string>Manos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPose">
            <property name="text">
             <string>Poses</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMascara">
            <property name="text">
             <string>MÃ¡scara de Cara</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMovimientos">
            <property name="text">
             <string>Movimientos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCaracteristicas">
            <property name="text">
             <string>Caracteristicas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDistancias">
            <property name="text">
             <string>Distancias</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnProfundidad">
            <property name="text">
             <string>Profundidad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 75 bold 16pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Extras:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPiedraPapelTijeras">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Piedra Papel Tijeras</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEntornoVirtual">
            <property name="text">
             <string>Entorno Virtual</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnVolumen">
            <property name="text">
             <string>Control de Volumen</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblNombre">
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 75 italic 11pt &quot;Calibri&quot;;
text-decoration: underline;
color:rgb(255, 255, 255);
border-radius:3px;
}</string>
            </property>
            <property name="text">
             <string>Dante Zulli 7mo 2da</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frContenedor">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(48, 24, 98, 255), stop:1 rgba(8, 20, 80, 255));
}

QLabel{
font: 10pt &quot;Calibri&quot;;
color:white;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frFoto">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lblFoto">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>GUI_Imagenes/fondoApp.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>10</number>
            </property>
            <property name="midLineWidth">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblNota">
            <property name="styleSheet">
             <string notr="true">background-color:rgba(0, 0, 0, 100)</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-style:italic;&quot;&gt;&amp;quot;Posicione el cursor sobre los botones del menÃº lateral para recibir&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-style:italic;&quot;&gt;informaciÃ³n sobre los mismos.&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
